presto-orc/src/main/java/com/facebook/presto/orc/metadata/Footer.java

```diff
@@ -71,6 +87,15 @@ public String toString()
                 .add("stripes", stripes)
                 .add("types", types)
                 .add("columnStatistics", fileStats)
+                .add("userMetadata", getUserMetadataAsString())
                 .toString();
     }
+
+    private String getUserMetadataAsString()
+    {
+        return userMetadata.entrySet().stream().collect(toMap(
+                Map.Entry::getKey,
+                entry -> entry.getValue().toStringUtf8()
```

Comment:
That was also my concern.
See discussion with @dain here: https://github.com/prestodb/presto/pull/5135#discussion-diff-61948727.

Originally I just added userMetadata to `toStringHelper`, which would use `Slice.toString` for map values.
@dain pointed that this is not very helpful but using UTF-8 has other issues. 
I would go for `2` and just list keys.